go/vt/vttablet/tabletmanager/vreplication/vplayer.go

```diff
@@ -125,7 +125,7 @@ func newVPlayer(vr *vreplicator, settings binlogplayer.VRSettings, copyState map
 		settings.StopPos = pausePos
 		saveStop = false
 	}
-
+	log.Infof("Starting VReplication player id: %v, startPos: %v, stop: %v, filter: %v", vr.id, settings.StartPos, settings.StopPos, vr.source)
```

Comment:
Do we want to print vr.source.Filter instead? Either way, probably worth `%+v` to make it easier to read.